### DISCUSSION TITLE
The `errors` object for invalid data is always empty

### DISCUSSION THREAD

**Discussion by Quidge** [react-hook-form/react-hook-form#3521] [Q&A] [answered]
It's a fairly complex project so I'm looking more for pointers on why this could be happening rather than give examples. We recently made the version bump from 5x to 6x. One of the changes that we had to make for the 5x to 6x jump is moving `validationSchema` to `resolver`.

We use yup as our validation library. Now, the invalidation errors that were previously coming back through `errors` from the `useFormContext` hook are no longer available. The `errors` object itself is always empty.

Ideas for where I should be looking here? Maybe my inputs aren't being registered correctly?

Also, I find this confusing in the documentation:

`https://react-hook-form.com/api/#useFormContext` cites wrapping the form in `FormContext` but the example uses `FormProvider`.

**Comment by Quidge** [comment, marked as answer]
Solved. Wasn't the version bump change. I don't understand how it was working before but the issue was due to error from code in my org. 👍

**Reply by bluebill1049** [reply]
sweet.
